### ISSUE TITLE
Learn Generics

### ISSUE THREAD

**Issue by gabbard** [isi-vista/adam#737]
Assigned to @mitchmarcus to write up a plan and reassign to @denizbeser for comments.

**Comment by gabbard** [comment]
@denizbeser : Can you remind me the status of this work?

**Comment by denizbeser** [comment]
@gabbard This is contingent on the plural learner, which we postponed last week after finding errors in the new learner in scenes with plural object. Migrating plural learner and then implementing learning generics was next on my to do list, and I can get started if those errors are resolved.

**Comment by LichMaster98** [comment]
While we're probably not entirely accurately learning these generic statements (we should be missing the 'plurality' information) our current verb learner is capable of at least learning something from these statements.
[m13-generics.zip](https://github.com/isi-vista/adam/files/4914731/m13-generics.zip)
